README.md

```diff
@@ -354,9 +354,16 @@ The ROOTPWA repository uses the master branch for development, meaning that the
 
 ### Commit Message Format ###
 
-ROOTPWA follows a policy for the commit messages, which is close to the standard git commit message policy. It consists of the following rules:
+ROOTPWA follows a policy for the commit messages, which is close to the standard git commit-message policy. It consists of the following rules:
 
-- The commit message must consist of at least three lines
+- The commit message must consist of at least three lines.
 - The first line of the commit message must not be longer than 80 characters.
-- The second line of the commit message must be empty
+- The second line of the commit message must be empty.
 - All remaining lines (of which there must be at least one) must not be longer than 80 characters.
+
+Please note that the ROOTPWA git repository uses a server hook to enforce this policy. Should any of your commits you would like to see integrated not fulfill these requirements, all of them will be rejected and you will have to re-edit the problematic messages. To reduce such problems, there is a git hook in the repository, which you can use on the client side to check all the commit messages already when committing to your repository. You can install it by running
```

Comment:
Ah, I forgot about this. Removed paragraph in e53c9dd7d213e51e129e13a7c8021e3505acc5b3.